mlc_llm/transform/fuse_split_rotary_embedding.py

```diff
@@ -151,6 +151,9 @@ def split_rotary(
 def fuse_split_rotary_embedding(
     mod, num_query_heads, num_kv_heads, hidden_size, position_embedding_base
 ):
+    if "rotary_embedding1" not in [gv.name_hint for gv in mod.functions]:
```

Comment:
What's `rotary_embedding1`?

Reply:
It applies the rotary embedding to Q/K in the single sequence flow. This pass hardcodes the name `rotary_embedding1` and looks it up in the IRModule, while in batching settings the RoPE application is fused in the attention kernel.

So here we check if `rotary_embedding1` exists in the IRModule, and return if it does not.